services/director/src/simcore_service_director/producer.py

```diff
@@ -186,6 +191,11 @@ class ServiceState(Enum):
         log.exception("Could not find swarm network")
 
     log.debug("Converted labels to docker runtime parameters: %s", docker_params)
+
+    # set labels for CPU and Memory limits
+    container_spec["Labels"]["nano_cpus"] = str(docker_params["task_template"]["Resources"]["Limits"]["NanoCPUs"])
+    container_spec["Labels"]["mem_limit"] = str(docker_params["task_template"]["Resources"]["Limits"]["MemoryBytes"])
```

Comment:
could we name the label with something like ```lmit_``` in front? just for clarity.

Reply:
done